com.github.torikulhabib.niki.yml

```diff
@@ -0,0 +1,60 @@
+app-id: com.github.torikulhabib.niki
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: com.github.torikulhabib.niki
+finish-args:
+  - '--share=network'
+  - '--socket=session-bus'
```

Comment:
You need to figure what dbus name are need. Addind the `session-bus` is usually not an option.